pkg/serve/index.html

```diff
@@ -381,6 +405,33 @@ <h1 class="title">Skenario</h1>
                             title: "Response Time (ms)"
                         }
                     }
+                },
+                {
+                    height: 500,
+                    width: chartWidth,
+                    data: {name: "cpu_utilizations"},
+                    transform: [
+                        {calculate: "datum.cpu_utilization", as: "cpu_utilization"},
+                        {calculate: "datum.calculated_at / 1000000000", as: "calculated_at_sec"}
+                    ],
```

Comment:
A minor comment.  But it would be easier to review if we separated out the UI changes from the backend changes.  You don't have to change it here, but it would good to do next time.

Reply:
ok, do it next time.